Modules/ThirdParty/HDF5/src/itkhdf5/src/itk_hdf5_mangle.h

```diff
@@ -2147,71 +1820,50 @@ e.g. remove mangling for H5Fget_info but leave it for H5Fget_info1 and H5Fget_in
 #define H5O__attr_reset itk_H5O__attr_reset
 #define H5O__attr_update_shared itk_H5O__attr_update_shared
 #define H5O__attr_write itk_H5O__attr_write
+#define H5O__check_msg_marked_test itk_H5O__check_msg_marked_test
 #define H5O__chunk_add itk_H5O__chunk_add
 #define H5O__chunk_delete itk_H5O__chunk_delete
 #define H5O__chunk_dest itk_H5O__chunk_dest
 #define H5O__chunk_protect itk_H5O__chunk_protect
+#define H5O__chunk_resize itk_H5O__chunk_resize
 #define H5O__chunk_unprotect itk_H5O__chunk_unprotect
 #define H5O__chunk_update_idx itk_H5O__chunk_update_idx
-#define H5O__create_link itk_H5O__create_link
+#define H5O__create_ohdr itk_H5O__create_ohdr
 #define H5O__dec_rc itk_H5O__dec_rc
 #define H5O__delete_mesg itk_H5O__delete_mesg
-#define H5O__exists_by_name itk_H5O__exists_by_name
+#define H5O__expunge_chunks_test itk_H5O__expunge_chunks_test
 #define H5O__free itk_H5O__free
-#define H5O__get_comment_by_name itk_H5O__get_comment_by_name
 #define H5O__get_info_by_idx itk_H5O__get_info_by_idx
-#define H5O__get_info_by_name itk_H5O__get_info_by_name
+#define H5O__get_rc_test itk_H5O__get_rc_test
 #define H5O__inc_rc itk_H5O__inc_rc
 #define H5O__init_package itk_H5O__init_package
-#define H5O__link itk_H5O__link
+#define H5O__is_attr_dense_test itk_H5O__is_attr_dense_test
+#define H5O__is_attr_empty_test itk_H5O__is_attr_empty_test
 #define H5O__link_oh itk_H5O__link_oh
 #define H5O__msg_alloc itk_H5O__msg_alloc
 #define H5O__msg_append_real itk_H5O__msg_append_real
 #define H5O__msg_copy_file itk_H5O__msg_copy_file
 #define H5O__msg_free_mesg itk_H5O__msg_free_mesg
 #define H5O__msg_free_real itk_H5O__msg_free_real
+#define H5O__msg_get_chunkno_test itk_H5O__msg_get_chunkno_test
 #define H5O__msg_iterate_real itk_H5O__msg_iterate_real
+#define H5O__msg_move_to_new_chunk_test itk_H5O__msg_move_to_new_chunk_test
 #define H5O__msg_remove_real itk_H5O__msg_remove_real
 #define H5O__msg_write_real itk_H5O__msg_write_real
+#define H5O__num_attrs_test itk_H5O__num_attrs_test
 #define H5O__obj_class itk_H5O__obj_class
 #define H5O__open_by_addr itk_H5O__open_by_addr
 #define H5O__open_by_idx itk_H5O__open_by_idx
 #define H5O__open_by_loc itk_H5O__open_by_loc
-#define H5O__open_name itk_H5O__open_name
```

Comment:
This commit removes a suspiciously large number of function names. Was there some big cleanup of HDF5 from version 1.10.6 to 1.10.7? If not, I would not remove the symbols unrelated to new additions/modifications.

Reply:
I think this is expected/correct.  I've just downloaded and diffed 1.10.6 vs 1.10.7 and indeed there are a huge number of changes. Their versioning scheme is not the usual one (where the 3rd number indicates a minor/patch release).